lib/tezos-client-unit-test

```diff
@@ -67,7 +67,10 @@ fix_address() {
     export ORIGINATION_SUBS="$TEMP_DIR/origination_subs"
     export ALL_SUBS="$TEMP_DIR/subs"
 
-    if ! grep -o '@Address([^)"]*)' "$test" | sort | uniq > "$FOUND_ADDRESSES"; then
+    # -o - only matching
+    # -E - extended matches
+    # -e - pattern follows this option
+    if ! grep -o -E -e '"KT1[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{33}"' "$test" | tr -d '"' | sort | uniq > "$FOUND_ADDRESSES"; then
```

Comment:
```suggestion
    # This is the bitcoin base 58 alphabet, with "I", "O", "0" and "l" removed.
    if ! grep --only-matching --extended-regexp '"KT1[123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{33}"' "$test" \
            | tr -d '"' | sort | uniq > "$FOUND_ADDRESSES"; then
```

Reply:
We can remove the `if` by touching the file first, and then running the grep.

Reply:
Addressed in later commit.